core_impl/src/main/java/io/opencensus/stats/MutableAggregation.java

```diff
@@ -33,6 +39,20 @@ private MutableAggregation() {
   abstract void add(double value);
 
   /**
+   * Update the summary stats by removing an expired value.
+   *
+   * @param value the expired value to be removed from population.
+   */
+  abstract void remove(double value);
```

Comment:
That sounds good. I've marked that Interval stats are not supported for stddev, mean, and range MutableAggregation.